Documentation/cmdref/cilium-agent.md

```diff
@@ -227,7 +227,7 @@ cilium-agent [flags]
       --k8s-service-proxy-name string                           Value of K8s service-proxy-name label for which Cilium handles the services (empty = all services without service.kubernetes.io/service-proxy-name label)
       --k8s-watcher-endpoint-selector string                    K8s endpoint watcher will watch for these k8s endpoints (default "metadata.name!=kube-scheduler,metadata.name!=kube-controller-manager,metadata.name!=etcd-operator,metadata.name!=gcp-controller-manager")
       --keep-config                                             When restoring state, keeps containers' configuration in place
-      --kube-proxy-replacement string                           Enable only selected features (will panic if any selected feature cannot be enabled) ("partial"), or enable all features (will panic if any feature cannot be enabled) ("strict"), or completely disable it (ignores any selected feature) ("disabled") (default "partial")
+      --kube-proxy-replacement string                           Enable only selected features (will panic if any selected feature cannot be enabled) ("false"), or enable all features (will panic if any feature cannot be enabled) ("true") (default "false")
```

Comment:
"true" and "false" don't really seem to be the right words to use. Why not just keep "partial" and "strict"?

Reply:
The problem is that the new `false` is not the same as the old `partial`. The change is that with false, host-port, external-ips, etc. need to be explicitly enabled, while with `partial` they were enabled by default.

Language-wise, `strict` and `partial` take much more explaining than `true` and `false`. Moreover, `false` here should be read to mean "full kube-proxy replecement has not been enabled"?